docs/replication-retry.md

```diff
@@ -0,0 +1,134 @@
+# Replication Retry
+
+## Description
+
+This feature offers a way for users to monitor and retry failed replication
+operations. It enables users to retrieve a list of failed operations and retry
+specific replication operations.
+
+## Requirements
+
+* The listing of failed operations.
+* The retry of a specific failed operation.
+
+## Design
+
+A RESTful API will expose methods for users to list and retry failed operations.
+See [Explanations](#explanations) for more detail.
+
+## Definition of API
+
+* GET `/_/crr/list/failed/all`
+
+    This GET request retrieves a listing of all failed operations. This
+    operation is useful if you're interested in whether any replication
+    operations have failed and want the entire listing.
+
+    Response:
+
+    ```sh
+    [{
+        bucket: <bucket>,
+        key: <key>,
+        versionId: <versionId>,
+        site: <site>
+    }]
+    ```
+
+* GET `/_/crr/list/failed/<bucket>/<key>/<versionId>`
+
+    This GET request retrieves a listing of all failed operations for a specific
+    object version. This operation is useful if you're interested in monitoring
+    the replication status of a specific object.
+
+    Response:
+
+    ```sh
+    [{
+        bucket: <bucket>,
+        key: <key>,
+        versionId: <versionId>,
+        site: <site>
+    }]
+    ```
+
+* POST `/_/crr/retry`
```

Comment:
For consistency with the GET, maybe use the same prefix: `/_/crr/failed/retry` or `/_/crr/failed?op=retry`

Reply:
The second form stays compatible with using a GET scheme like `/_/crr/failed/bucket/object/versionId`

Reply:
Will update to `/_/crr/failed/retry`.